portfolio/src/main/java/com/google/sps/servlets/InteractiveChartServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** TODO(briafassler): store data to a database */
+@WebServlet("/interactive-chart")
+public class InteractiveChartServlet extends HttpServlet {
+  private Map<String, Integer> emotionVotes = new HashMap<>();
+  private static String EMOTION_DATA_PARAM = "emotion";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Gson gson = new Gson();
+    String json = gson.toJson(emotionVotes);
+    response.setContentType("application/json");
+    response.getWriter().println(json);
```

Comment:
you can do this in about one line with print(new Gson().toJson()).

not really that important but if you're looking for a few more efficient lines that's a good approach, if not up to you

Reply:
done